composer.json

```diff
@@ -12,15 +12,15 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
+        "php": "^7.1",
```

Comment:
@dbu as we are bound to the latest `phpcr-bundle` – fixes to fix deprecation warnings are done on its master – we have to follow its php version constraints.